templates/availablerooms.html

```diff
@@ -227,13 +284,12 @@ <h2 style="font-weight:700;">Rooms</h2>
             <option value="First College">First</option>
             <option value="Forbes College">Forbes</option>
             <option value="Mathey College">Mathey</option>
-            <option value="No Res College Associated">No Res College Associated</option>
-            <option value="New College East">New College East</option>
             <option value="New College West">New College West</option>
+            <option value="No Res College Associated">No Res College Associated</option>
             <option value="Rockefeller College">Rocky</option>
             <option value="Whitman College">Whitman</option>
             <option value="Upperclass">Upperclassman</option>
-
+            <option value="Yeh">Yeh</option>
```

Comment:
ye

Reply:
Yeh is correct?

Reply:
Yes, New College East was renamed to [Yeh](https://www.dailyprincetonian.com/article/2022/05/yeh-college-new-college-east-princeton-residential-building-student-housing)

Reply:
done